mitodl/mitxonline
# Removing feature flag for program UI; small styling change to My Courses tab

#### Pre-Flight checklist


#### What are the relevant tickets?

Closes #1310 
Closes #1290

#### What's this PR do?

Removes the `FEATURE_ENABLE_PROGRAM_UI` flag. Also fixes the dashboard's `My Courses` tab's styling (it was a bit off from what you get if there are program enrollments). 

#### How should this be manually tested?

Ensure you have a learner account that has a program enrollment (either implicit or explicit). You should see the program UI regardless of whether or not you have `FEATURE_ENABLE_PROGRAM_UI` set in your `.env` file or if you've added `?enable_programs` to the URL. 

If your learner does not have program enrollments (meaning, either an explicit enrollment in a program or an enrollment in a course that's in a program and no explicit unenrollments in the program), they should only see the My Courses tab per #1303
